main.js

```diff
@@ -44,6 +44,26 @@ const handlers = {
     });
   },
 
+  handleLoadSpec: () => {
+    const options = {
+      properties: ['openFile'],
+      filters: [
+        {
+          name: 'Vega Lite Specs',
+          extensions: ['json'],
+        },
+      ],
+    };
+
+    dialog.showOpenDialog(mainWindow, options, (filenames) => {
+      if (filenames && filenames.length > 0) {
```

Comment:
It can only open one file but i believe an empty array comes back if the user clicks cancel.